spec/features/trainer_spec.rb

```diff
@@ -14,11 +14,10 @@
 
   describe "training with some cards" do
     before do
-      2.times do
-        card = create(:card)
-        card.update_attribute(:review_date, Time.now)
-        visit root_path
-      end
+      user = create(:user_with_cards, cards_count: 2)
+      user.cards.each { |card| card.update_attribute(:review_date,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.